source/includes/_reporting-entities.md

```diff
@@ -7,7 +7,21 @@
 </aside>
 
 To submit data for another entity, you need to be invited as a contributor, by
-an account manager. For testing purposes, you can create a reporting entity,
+an account manager. 
+
+##Providing your Application with access to a Reporting Entity
+
+### 1.  Go to the GRESB Portal and click on access
+<img src="/Users/jaikishpai/api-docs/source/images/access_permission_pictures/access_link.png" alt="access_link pic">
```

Comment:
you sure this will be accessible to users on the internet? Shouldn't we store it in a public S3 bucket in AWS?

Reply:
good catch! I have updated the relative path to the images. So the path will be `<img src="/images/access_permission_pictures/access_link.png" alt="access_link pic">` the image will be picked up from the images folder part of the project.